packages/@tinacms/api-git/src/file-writer.test.ts

```diff
@@ -0,0 +1,13 @@
+import { checkFilePathIsInGitRepo } from './file-writer'
+
+test('Check path if is in git repo - bad', () => {
```

Comment:
I would wrap these tests with a `describe` to make the output more readable:

```ts
describe("checkFilePathIsInGitRepo", () => {
  test('returns false if path not in repo', () => {
    // ...
  })

  test('returns true if path in repo', () => {
    // ...
  })
})
```

Reply:
ok, I don't have experience with jest this is looks like mocha ;-)